src/multibody/joint/joint-base.hpp

```diff
@@ -336,6 +336,23 @@ namespace se3
     ConfigVector_t neutralConfiguration() const
     { return derived().neutralConfiguration_impl(); } 
 
+    /**
+     * @brief      Normalize a configuration
+     *
+     * @param[in]  q     Configuration to normalize (size full model.nq)
+     *
+     * @return     The normalized configuration
+     */
+    ConfigVector_t normalized(const Eigen::VectorXd & q) const
+    { return derived().normalized_impl(q); }
+
+    /**
+     * @brief      Default implementation of normalized
+     */
+    ConfigVector_t normalized_impl(const Eigen::VectorXd & q) const
```

Comment:
This kind of default behavior __can__ be dangerous if someone adds a new type of derived joint that has its particular behavior but do not reimplement the method.
In the current use of the CRTP interface it would lead  to a compilation error as method_impl does not exist in the derived joint, except for this method

It can also be noted that joints with 1 DoF can often be factorized in a default behavior mothod